content/docs/other-guides/freq-ask-questions.md

```diff
@@ -0,0 +1,12 @@
++++
+title = "Frequently asked questions"
+description = ""
+weight = 110
++++
+
+
+### Does Kubeflow Support Helm?
+
+Yes Kubeflow support Helm. You don't need to use ksonnet to use TFServing, TFJob and others. You can define those resources using whatever K8s tooling you want; raw YAML, Helm, Kustomize, Ksonnet.
```

Comment:
Consider taking another pass at this paragraph. Some hints:

- `Yes Kubeflow support Helm.` > `Yes, Kubeflow supports Helm.`
- See if you can find a way to shorten the second sentence, and try to cut down on word repetition. What are people using these other tools for? Extra context could be handy.
- Adding product links might help people to discover the other tools mentioned in this paragraph.
- `K8s` > `Kubernetes`.
- The semicolon seems out of place here. A full colon might be more appropriate, but it depends on how much you rewrite this.
- Inconsistency in capitalization of "ksonnet". Check what the canonical capitalization is, and apply across the board.